lib/converterbase.rb

```diff
@@ -934,14 +934,22 @@ def to_ruby(match, m1, m2, openclose_symbols)
       else
         # なろうのルビ対象文字を辿って｜を挿入する（青空文庫となろうのルビ仕様の差異吸収のため）
         # 空白もルビ対象文字に含むのはなろうの仕様である
-        m1.sub(/([#{CHARACTER_OF_RUBY} 　]+)$/) {
-          match_target = $1
-          if match_target =~ /^(　+)/
-            "#{$1}［＃ルビ用縦線］#{match_target[$1.length..-1]}"
+        if m2 =~ /^([ぁ-んァ-ヶーゝゞ・]+)[ 　]?([ぁ-んァ-ヶーゝゞ・]*)$/
+          f1, f2 = $1, $2
+          if m1 =~ /([#{CHARACTER_OF_RUBY}]+)([ 　])([#{CHARACTER_OF_RUBY}]+)$/
+            m1.sub(/([#{CHARACTER_OF_RUBY}]+)([ 　])([#{CHARACTER_OF_RUBY}]+)$/) {
+              if f2 == ""
+                "#{$1}#{$2}［＃ルビ用縦線］#{$3}《#{ruby_youon_to_big(m2)}》"
```

Comment:
__[rubocop]__
* Metrics/BlockNesting: Avoid more than 3 levels of block nesting. [:ref](https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count)

<!-- Comment by [SideCI](https://sideci.com) -->